HijackSession appears to be misnamed and breaks at compile time

```
$ go get ./...
# github.com/whyrusleeping/hellabot
../../whyrusleeping/hellabot/hellabot.go:211:15: bot.hijackSession undefined (type *Bot has no field or method hijackSession, but does have HijackSession)
../../whyrusleeping/hellabot/recon_darwin.go:6:6: type Bot has both field and method named HijackSession
```

```
$ go version
go version go1.10 darwin/amd64
```

--- a/recon_darwin.go
+++ b/recon_darwin.go
@@ -3,6 +3,6 @@ package hbot
 func (irc *Bot) StartUnixListener() {}
 
 // Attempt to hijack session previously running bot
-func (irc *Bot) HijackSession() bool {
+func (irc *Bot) hijackSession() bool {
 	return false
 }

--- a/recon_windows.go
+++ b/recon_windows.go
@@ -3,6 +3,6 @@ package hbot
 func (irc *Bot) StartUnixListener() {}
 
 // Attempt to hijack session previously running bot
-func (irc *Bot) HijackSession() bool {
+func (irc *Bot) hijackSession() bool {
 	return false
 }

